curriculum/challenges/english/08-coding-interview-prep/rosetta-code/left-factorials.md

```diff
@@ -0,0 +1,88 @@
+---
+id: 5a23c84252665b21eecc7ee0
+title: Left factorials
+challengeType: 5
+---
+
+## Description
+<section id='description'>
+<b>Left factorials</b>,  $ !n $,  may refer to either  <i>subfactorials</i>  or to  <i>factorial sums</i>. The same notation can be confusingly seen used for the two different definitions. Sometimes,  <i>subfactorials</i>  (also known as <i>derangements</i>)  may use any of the notations:
+$$!n`$$
+$$!n$$
+$$n¡$$
+(It may not be visually obvious, but the last example uses an upside-down exclamation mark.) This task will be using this formula for <b>left factorial</b>:
+$$ !n = \sum_{k=0}^{n-1} k! $$
+where $!0 = 0$
+Write a function to calculate the left factorial of a given number.
+</section>
+
+## Instructions
```

Comment:
```
<section id='instructions'>
Write a function to calculate the left factorial of a given number.
 </section>
```

Reply:
I believe this is fixed, marking as resolved